apps/admin_app/lib/admin_app_web/mailer/order_export_mail.ex

```diff
@@ -0,0 +1,20 @@
+defmodule AdminAppWeb.OrderExportMail do
+  @moduledoc """
+  Composing email to send order export.
+  """
+
+  import Swoosh.Email
+  alias AdminAppWeb.Mailer
+
+  def order_export_mail(attachment, user, type) do
+    sender_email = Application.get_env(:admin_app, AdminAppWeb.Endpoint)[:sendgrid_sender_mail]
+
+    new()
+    |> to(user.email)
+    |> from({"Aviacommerce", sender_email})
```

Comment:
I think here store name should be there.

Reply:
makes sense